exec/java-exec/src/test/java/org/apache/drill/exec/ExecTest.java

```diff
@@ -120,4 +124,27 @@ protected LogicalExpression parseExpr(String expr) throws RecognitionException {
     return ret.e;
   }
 
+  /**
+   * mock current locale to US
+   */
+  public static void mockUSLocale() {
+    new MockUp<DateTimeUtils>() {
+      @Mock
+      public DateFormatSymbols getDateFormatSymbols(Locale locale) {
+        return new DateFormatSymbols(Locale.US);
+      }
+    };
+  }
+
+  /**
+   * mock current timezone to UTC
```

Comment:
Please start the sentence with a capital letter and add a description about the method which is mocked.